include/ebpf_registry_helper.h

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <ntddk.h>
+
+#define REG_CREATE_FLAGS 0
+#define GUID_STRING_LENGTH 38 // not inlcuding the null terminator.
+
+typedef struct _ebpf_registry_key
```

Comment:
why a struct with only one field? Did you mean to typedef?

Reply:
Fixed.